java/src/main/java/com/databricks/mlflow/mleap/MLeapLoader.java

```diff
@@ -0,0 +1,50 @@
+package com.databricks.mlflow.mleap;
+
+import com.databricks.mlflow.LoaderModule;
+import com.databricks.mlflow.models.Model;
+import com.databricks.mlflow.sagemaker.MLeapPredictor;
+import com.databricks.mlflow.sagemaker.Predictor;
+import com.databricks.mlflow.sagemaker.PredictorLoadingException;
+import com.databricks.mlflow.utils.FileUtils;
+
+import java.util.Optional;
```

Comment:
[import order: all non-static imports in a single block](https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing)